### PR TITLE
Add OSX support for shell commands

### PR DESCRIPTION
**Pull request by weslly**
They work just like linux, so it was just a copy&paste thing. I've tested it and everything seems ok.

### CODE REVIEW

**Comment by guillermooo** [comment]
Thanks for the PR. Wouldn't it make more sense to have a separate **vintageex_osx_[terminal|shell]** pair of settings for OS X?

**Comment by weslly** [comment]
Fixed :)

File: plat/osx.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import os
+import subprocess
+
+
+def run_and_wait(view, cmd):
+    term = os.path.expandvars("$COLORTERM") or os.path.expandvars("$TERM")
```
Comment (guillermooo) [inline plat/osx.py]: Here the user preferences should be consulted for **vintageex_osx_terminal**. I think this is missing from the linux counterpart too. Update this if you want or I'll do it myself after the merge when I get around to doing it.

Comment (weslly) [inline plat/osx.py]: updated both

**Comment by guillermooo** [comment]
Thanks!

**Pull request merged**